### PR TITLE
feat(wallet_delete): Remove existing wallet

### PR DESCRIPTION
**Pull request by ssakone**
Remove existing wallet.

This pull request include two new feature, 
- getText : User input dialog and return value entered
- showText : Display a message dialog to screen 
These two new features can be improved as needed and can be used to replace the different confirmation dialog boxes that already exist in the application.

Need to test

- [x] input dialog display when user click on delete icon in startup
- [x] password work correctly
- [x] validate and check if its work
- [x] last message dialog display well.
- [x] No overflow. 

@smk762  maybe different text need to be changed.

image
image
image

### CODE REVIEW

**Review by Milerius — COMMENTED** [review]
Works correctly on osx, will test shortly on linux.

**Comment by smk762** [comment]
Unclear here wheat is intended to go into the input. It's asking "Do I want to delete wallet?" I enter "yes" and it tells me that is not my correct password.

image

Change `do you want to delete this wallet` to `Enter password to confirm wallet delete`
Also for the title, change `Delete wallet` to `Delete {wallet_name} wallet?`

On the next screen, after entering password
image

change `{wallet_name} Wallet deleted successfully` to `{wallet_name} wallet deleted successfully` (lowercase  `w`)

**Comment by ssakone** [comment]
@smk762  still working on correct field text size.

File: atomic_defi_design/qml/Screens/FirstLaunch.qml [hunk 1]
```diff
@@ -336,8 +336,8 @@ SetupPage {
                                 onClicked: {
                                     let wallet_name = model.modelData
                                     let dialog = app.getText({
-                                        "title": qsTr("Delete Wallet"),
-                                        text: qsTr("Do you want to delete this wallet"),
+                                        "title": qsTr("Delete") + " %1 ".arg(wallet_name) + ("wallet?"),
+                                        text: qsTr("Enter password to confirm wallet delete"),
```
Comment (Milerius) [inline atomic_defi_design/qml/Screens/FirstLaunch.qml:340]: "Enter password to confirm the deletion of the wallet" is probably more accurate, cc @smk762  if you have any other idea ?

Comment (smk762) [reply atomic_defi_design/qml/Screens/FirstLaunch.qml:340]: yeah, that's better :+1:
Could reduce to `Enter password to confirm deletion of {wallet_name} wallet`

**Comment by tonymorony** [comment]
needs to be synced with dev (https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/1097) for CI build pass

**Review by smk762 — APPROVED** [review]

**Pull request merged**